.pre-commit-config.yaml

```diff
@@ -18,14 +18,14 @@ repos:
     hooks:
       - id: python-check-blanket-noqa
 
-  - repo: https://github.com/pre-commit/mirrors-isort
```

Comment:
FYI, https://github.com/pre-commit/mirrors-isort is deprecated hence this change.